tests/exception_report/test_large_exception.py

```diff
@@ -0,0 +1,51 @@
+import sys
+
+from django.test import TestCase
+from django.test.client import RequestFactory
+from django.views.debug import ExceptionReporter
+
+
+class ExceptionReport(TestCase):
+    factory = RequestFactory()
+
+    def test_large_sizable_object(self):
+        lg = list(range(50 * 1024 * 1024))
```

Comment:
Do we need 50 million elements?

Reply:
I was just trying to check the efficiency of library's able to handle extreme sizes of objects and rendering them in a reasonable time. 😅What could be a more optimal value?